### PR TITLE
fix engrosh

### PR DESCRIPTION
**Pull request by bendnuts**
engrosh is gud noe

### CODE REVIEW

File: docs/modding/4-spriting.md [hunk 1]
```diff
@@ -36,23 +36,23 @@ It is highly recommended that you use spriting software that supports transparen
       - Pretty bloated for spriting in Mindustry style.
 
   5. **[Paint.NET](https://www.getpaint.net/)**
-      - Very basic painting software, not to be confused with Paint 3D, Paint&#46;NET is usable but not as convenient as the above mentioned.
-      - Paint&#46;NET lacks basic features used for spriting in mindustry style. However, you can get some of these missing features with plugins that you can install manually.
```
Comment (Eschatologue) [inline docs/modding/4-spriting.md:40]: There's a reason that I did this, markdown would convert regular Paint.NET to a link,

Comment (bendnuts) [reply docs/modding/4-spriting.md:40]: oh?

**Comment by bendnuts** [comment]
bruh the &\#46 really fucked up

**Comment by Anuken** [comment]
Conflicts after merge.

**Comment by bendnuts** [comment]
hmm

**Comment by bendnuts** [comment]
> Conflicts after merge.

fixed on #da6b6f3

**Comment by bendnuts** [comment]
half fucked up my resolutioning

**Pull request closed without merging**